norns/maiden.md

```diff
@@ -70,38 +70,49 @@ Each entry has an **INSTALL** action, which can be used to install the selected
 You will have to restart norns if a freshly-installed project contains an engine
 {: .label}
 
-### contribute
+### contribute & gather
 
-To add to the [community project repo](https://github.com/monome/norns-community/blob/master/community.json), please submit a pull request with the following information:
+The norns ecosystem was created with community as a focus. The exchange of ideas leads to new ideas.
 
-```json
-    {
-      "project_name": "NAME",
-      "project_url": "URL",
-      "author": "NAME",
-      "description": "WORDS",
-      "discussion_url": "LINES_LINK",
-      "tags": ["TAG", "TAG", "TAG"],
-      "origin": "IF_APPLICABLE: lines"
-    },
-```
+As such, you are encourage to share your creations with the community.
+
+The first step is to host the sources of your script publicly, either as a git repository (e.g. on GitHub, GitLab ...) or web-hosted zip archive.
 
-Be sure you include information at the top of your script to help future users:
+Be sure you include a block of information at the top of your script. It will serve as a brief documentation when launching it from norns.
```

Comment:
small thing
>Be sure you include the following block of information at the top of your script. It will serve as a brief documentation when launching it from norns.